rosie/core/tests/test_invalid_cnpj_cpf_classifier.py

```diff
@@ -7,38 +7,21 @@
 
 
 class TestInvalidCnpjCpfClassifier(TestCase):
-
     def setUp(self):
```

Comment:
Hey @luizberti ! What do you think of use [setUpClass](https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUpClass) instead of run setUp for each test since it will be re used on all tests?

In the best world I would avoid a big fixture since each test just use a small piece of this fixture and avoid file access unnecessary. But for now just use setUpClass its already enough.

Reply:
If we remove the need of a fixture file, there will not be necessary opening a file in each test, so we could maintain everything in setUp.

Reply:
@luizberti I believe that you could use a approach similar with [this test](https://github.com/datasciencebr/rosie/pull/89/files#diff-3861458dcb40e69ca2d76d3d795045cfR14)